src/main/java/storage/WriteDataFile.java

```diff
@@ -0,0 +1,105 @@
+package storage;
+
+import cheatsheet.CheatSheet;
+
+import java.io.IOException;
+import java.io.FileWriter;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static storage.ParseCheatSheetFile.NAME;
+import static storage.ParseCheatSheetFile.PROGRAMMING_LANGUAGE;
+import static storage.ParseCheatSheetFile.DETAILS;
+
+/**
+ * Allows the user to write data based on the tasks currently present in the
+ * list of tasks into duke.txt.
+ */
+public class WriteDataFile extends DataFile {
+    private static final String FILE_EXTENSION = ".txt";
+
+    // WriteDataFile Constructor
+    public WriteDataFile() {
+        executeFunction();
+    }
+
+    /**
+     * Converts the tasks present in the list of tasks into
+     * a string. This string is then saved in a form of .txt file
+     * into documentTracker.txt.
+     */
+    @Override
+    public void executeFunction() {
+        storeCheatSheet();
+    }
+
+    /**
+     * Converts the tasks present in the list of tasks into
+     * a string.
+     */
+    private void storeCheatSheet() {
+        /*for(CheatSheet cheatSheet : cheatSheets) {
+            convertStringToFile(cheatSheet);
+        }
+        */
+    }
+
+    /**
+     * Converts the cheatSheet into a file format.
+     *
+     * @param cheatSheet The cheatSheet that is currently being converted into a file.
+     */
+    private void convertStringToFile(CheatSheet cheatSheet) {
+
```

Comment:
maybe remove the spacing between first line of the method and the next, for consistency